### PR TITLE
Bug 1184662 - Update Intern docs to include intern version

### PR DESCRIPTION
**Pull request by darkwing**
Since intern just upgraded, we should tell people to use the version we've been developing on

### CODE REVIEW

**Comment by openjck** [comment]
Mind if I grab this, @groovecoder?

**Comment by groovecoder** [comment]
:+1:

File: docs/tests-ui.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Installing Dependencies
 
 3. From the `tests/ui/` directory, use NPM or another package manager to install Intern::
 
-    npm install intern
+    npm install intern@2.2.2
```
Comment (openjck) [inline docs/tests-ui.rst]: We could alternatively recommend that people run `npm install intern@^2.0.0`. That will install the latest version in the 2.0.0 branch, which should be API compatible. That way we won't need to update the file for every version.

**Comment by darkwing** [comment]
John wins.  Updated.

**Pull request merged**